containers/Ideas/IdeasListItem.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { View, Text, TouchableOpacity } from 'react-native';
+import { styles } from './IdeasListItem.styles';
+
+import { ButtonCounter } from '../../components/buttonCounter';
+
+export const IdeasListItem = (props) => {
+  console.log(props);
```

Comment:
Could you remove this?

Reply:
ahh, thanks..maybe i'll add another eslint rule.